src/main/java/com/pmm/budgetcalculator/expensing/repositories/BaseEntity.java

```diff
@@ -1,4 +1,4 @@
-package com.pmm.budgetcalculator.foundation;
+package com.pmm.budgetcalculator.expensing.repositories;
```

Comment:
Model/entity classes hould not be in repositories package. Also try to name packages singular